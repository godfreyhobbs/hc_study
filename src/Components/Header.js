import React, { Component } from 'react';
import logo from '../logo.svg';

class MainHeader extends Component {
  render() {
    return (
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <h2>Welcome to the Viel Cornwell Hookah Study</h2>
      </header>
    );
  }
}

export default MainHeader;
