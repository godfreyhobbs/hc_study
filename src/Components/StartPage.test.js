import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('renders correctly', () => {
  const tree = renderer
.create(<Header />)
   .toJSON();
expect(tree).toMatchSnapshot();
});
