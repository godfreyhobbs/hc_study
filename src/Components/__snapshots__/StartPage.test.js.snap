// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="App-title"
  >
    Welcome to the Viel Cornwell Hookah Study
  </h1>
</header>
`;
